data/README.md

```diff
@@ -10,17 +10,16 @@ This module holds a global state variable and exposes a "Redux-like" API contain
 
 If your module or plugin needs to store and manipulate client-side data, you'll have to register a "reducer" to do so. A reducer is a function taking the previous `state` and `action` and returns an update `state`. You can learn more about reducers on the [Redux Documentation](https://redux.js.org/docs/basics/Reducers.html)
 
-### `wp.data.getState()`
+This function returns a Redux-like store object with the following methods:
```

Comment:
`wp.data.registerReducer` - should it have name passed as a first param? I guess it should. I noticed it today, when reading docs.